Sphinxsetup.sh

```diff
@@ -10,11 +10,21 @@ fi
 DISTRIBUTION_ID=$(lsb_release -i -s)
 if [ ${DISTRIBUTION_ID} = 'Ubuntu' ]; then
   DISTRIBUTION_CODENAME=$(lsb_release -c -s)
-  if [ ${DISTRIBUTION_CODENAME} = 'focal' ] || [ ${DISTRIBUTION_CODENAME} = 'bionic' ]; then
-    sudo add-apt-repository universe
+  if [ ${DISTRIBUTION_CODENAME} = 'focal' ] || [ ${DISTRIBUTION_CODENAME} = 'bionic' ] || [ ${DISTRIBUTION_CODENAME} = 'lunar' ]; then
+    sudo add-apt-repository universe -y
   fi
 fi
 
+# create a Python venv on more recent releases:
+if [ ${DISTRIBUTION_CODENAME} == 'lunar' ]; then
+    sudo apt install python3.11-venv
```

Comment:
Why not `python3-venv`?

Reply:
Fixed. I didn't realize the default python packages were there. Looks like those are linked to whatever the distro is currently using.